### PR TITLE
338 betting strategy

### PR DESCRIPTION
**Pull request by gabrielfior**

### CODE REVIEW

File: prediction_market_agent/agents/think_thoroughly_agent/deploy.py
Comment (kongzii) [inline prediction_market_agent/agents/think_thoroughly_agent/deploy.py:46]: 👍 This new approach is cool

**Review by kongzii — APPROVED** [review]

**Comment by gabrielfior** [comment]
Waiting on PMAT's new release before merging this

**Pull request merged**
